quelpa.el

```diff
@@ -1839,6 +1852,7 @@ the global var `quelpa-upgrade-p' is set to nil."
   (when (quelpa-setup-p) ;if init fails we do nothing
     (let* ((quelpa-upgrade-p (if current-prefix-arg t quelpa-upgrade-p)) ;shadow `quelpa-upgrade-p'
            (quelpa-stable-p quelpa-stable-p) ;shadow `quelpa-stable-p'
+           (quelpa-force-upgrade-p quelpa-force-upgrade-p) ;shadow `quelpa-force-upgrade-p'
```

Comment:
Again, this variable seems a bit redundant with `quelpa-upgrade-p`, and shadowing it seems confusing.  Can this be simplified?